tests/components/whirlpool/test_sensor.py

```diff
@@ -147,9 +147,19 @@ async def test_washer_sensor_values(
     assert state.state == thetimestamp.isoformat()
 
     state_id = f"{entity_id.split('_')[0]}_detergent_level"
+    registry = entity_registry.async_get(hass)
+    entry = registry.async_get(state_id)
+    assert entry
+    assert entry.disabled
+    assert entry.disabled_by is entity_registry.RegistryEntryDisabler.INTEGRATION
+    update_entry = registry.async_update_entity(
+        entry.entity_id, **{"disabled_by": None}
+    )
+    await hass.async_block_till_done()
+    assert update_entry != entry
+    assert update_entry.disabled is False
     state = hass.states.get(state_id)
-    assert state is not None
-    assert state.state == "50"
+    assert state is None
```

Comment:
How was this 50 before but now is None? Don't see anything else that would change this value